shadowsocks/cipher.go

```diff
@@ -77,6 +77,16 @@ func newAesGCM(key []byte) (cipher.AEAD, error) {
 	return cipher.NewGCM(blk)
 }
 
+func maxCipherOverhead() int {
```

Comment:
I prefer the term "TagSize" for consistency with the Shadowsocks documentation and the rest of this code.

Reply:
Done